akka-docs/src/test/java/jdocs/stream/operators/SourceOrFlow.java

```diff
@@ -29,6 +34,12 @@ void logExample() {
     ;
   }
 
+  void zipWithIndexExample() {
+    //#zip-with-index
+    Source.from(Arrays.asList(7, 8, 9)).zipWithIndex();
```

Comment:
I think we can do better, for someone who doesn't already know `zipWithIndex` this doesn't show what it does. So maybe something like running it with a printline-foreach and having a comment (or text in the md) saying what will be printed. I think it's good to use something else than integers for the source as well, so that it is easy to distinguish which one is the index and which is the element from the source.
```
// will output pairs with ('a', 0), ('b', 1), ('c', 2)
```

Reply:
👍